api/client/flightClient 2.js

```diff
@@ -0,0 +1,13 @@
+const apiKey = require('./apikey')
+
+class FlightClient {
+  loadFlights(numberOfTravellers, outboundDestination, inboundDestination, departureDate, returnDate){
```

Comment:
Think we can delete this file rather than merging it with the main branch? I don't think it's needed anymore.